### PR TITLE
completed button check in/out functionality and page styling

### PR DESCRIPTION
**Pull request by kssampson**
- added functionality for check-in and pick-up
- styling for ReservationDetails page

### CODE REVIEW

File: src/components/valet-reservation/ReservationDetails.jsx [hunk 1]
```diff
@@ -22,59 +23,135 @@ const ReservationDetails = () => {
...
+              <span className="font-semibold">Date:</span> {reservation.date}
+            </p>
+          </div>
+          <div>
+            <p>
+              <span className="font-semibold">Parking Spot ID:</span>{' '}
+              {reservation.parking_spot_id}
+            </p>
+            <p>
+              <span className="font-semibold">Keybox Number:</span>{' '}
```
Comment (kjmak14) [inline src/components/valet-reservation/ReservationDetails.jsx:71]: Line 94 and 90 convey the same information. Maybe consolidate them

**Review by kjmak14 — APPROVED** [review]
Seems good, I'll refactor to redux later on

**Comment by kssampson** [comment]
moved the buttons into another component and conditionally rendered to show one button for checkin/checkout

**Review by stevkim — APPROVED** [review]
Looks great! We talked about doing this together

**Pull request merged**